src/components/AppSidebar/AppSidebar.vue

```diff
@@ -181,19 +175,17 @@ include a standard-header like it's used by the files app.
 						<!-- favourite icon -->
 						<div v-if="canStar || $slots['tertiary-actions']" class="app-sidebar-header__tertiary-actions">
 							<slot name="tertiary-actions">
-								<a v-if="canStar"
-									class="app-sidebar-header__star"
+								<Button v-if="canStar"
+									type="tertiary-no-background"
 									@click.prevent="toggleStarred">
-									<span v-if="starLoading" class="icon-loading-small" />
-									<Star v-else
-										:class="{
-											'star--starred': isStarred,
-											'star--star': !isStarred,
-										}"
-										class="star"
-										:size="20"
-										decorative />
-								</a>
+									<template #icon>
+										<span v-if="starLoading" class="icon-loading-small" />
+										<Star v-else
+											:fill-color="isStarred ? '#FC0' : 'currentColor'"
```

Comment:
The string `'currentColor'` is no valid color, no?

Reply:
It is passed directly to the svg fill attribute https://github.com/robcresswell/vue-material-design-icons/blob/1bc89fe0a7a01914c247263947a976b854d7c14c/build.ts#L20 and is the same as the default https://github.com/robcresswell/vue-material-design-icons/blob/1bc89fe0a7a01914c247263947a976b854d7c14c/build.ts#L41